lib/constants.js

```diff
@@ -9,14 +9,16 @@ module.exports = {
 
   LEVEL_KEY: 'level',
 
+  LEVEL_LABEL: 'levelLabel',
+
   TIMESTAMP_KEY: 'time',
 
   LEVELS: {
     default: 'USERLVL',
     60: 'FATAL',
     50: 'ERROR',
-    40: 'WARN ',
-    30: 'INFO ',
```

Comment:
This should be reverted. As pointed out, this was added for formatting via https://github.com/pinojs/pino-pretty/pull/47.

Reply:
I'm trying to still have the space removed and instead use `\t`.
However the tests are a bit painful to change since `t.is` is having a hard time comparing the actual vs expected versions, must be some unicode difference.

Reply:
Other that this is works fine:
```sh
[12:26:21] luis:pino-pretty-test $ node example.js | pino-pretty -c false -t -l
INFO	[2020-09-29 11:26:37.616 +0000] (83428 on nf.local): hello world
WARN	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): Warning...
ERROR	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): Oops, this is an error!
INFO	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): hello world
WARN	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): Warning...
ERROR	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): Oops, this is an error!
INFO	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): hello world
WARN	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): Warning...
ERROR	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): Oops, this is an error!
INFO	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): hello world
WARN	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): Warning...
ERROR	[2020-09-29 11:26:37.617 +0000] (83428 on nf.local): Oops, this is an error!
[12:26:37] luis:pino-pretty-test $ node example.js | pino-pretty -t -l
INFO	[2020-09-29 11:26:44.584 +0000] (83444 on nf.local): hello world
WARN	[2020-09-29 11:26:44.584 +0000] (83444 on nf.local): Warning...
ERROR	[2020-09-29 11:26:44.584 +0000] (83444 on nf.local): Oops, this is an error!
INFO	[2020-09-29 11:26:44.584 +0000] (83444 on nf.local): hello world
WARN	[2020-09-29 11:26:44.584 +0000] (83444 on nf.local): Warning...
ERROR	[2020-09-29 11:26:44.584 +0000] (83444 on nf.local): Oops, this is an error!
INFO	[2020-09-29 11:26:44.584 +0000] (83444 on nf.local): hello world
WARN	[2020-09-29 11:26:44.584 +0000] (83444 on nf.local): Warning...
ERROR	[2020-09-29 11:26:44.584 +0000] (83444 on nf.local): Oops, this is an error!
INFO	[2020-09-29 11:26:44.584 +0000] (83444 on nf.local): hello world
WARN	[2020-09-29 11:26:44.585 +0000] (83444 on nf.local): Warning...
ERROR	[2020-09-29 11:26:44.585 +0000] (83444 on nf.local): Oops, this is an error!
```

Reply:
What do you think @jsumners

Reply:
Please work on that in a separate issue if it is important to you.

Reply:
Ah, you're totally right
TY

Reply:
Done